src/main/java/edu/ie3/datamodel/io/naming/DatabaseNamingStrategy.java

```diff
@@ -42,6 +42,11 @@ public String getTimeSeriesPrefix() {
     return TIME_SERIES_PREFIX;
   }
 
+  /** Returns the String of the load profile table */
+  public String getLoadProfileTableName() {
+    return LOAD_PROFILE;
```

Comment:
is this what we want here?

Reply:
Since we have all load profiles in the same database table, we simply return the table name here.